app/bundles/EmailBundle/Translations/en_US/messages.ini

```diff
@@ -59,11 +59,11 @@ mautic.email.contact_already_received_marketing_email="%contact% has already rec
 mautic.email.contact_has_no_email="%contact% has no email address."
 mautic.email.builder.addcontent="Click to add content"
 mautic.email.builder.index="Extras"
-mautic.email.campaignId.filter="Campaign"
```

Comment:
Why is this lang string removed? It's used in several places. Example:

https://github.com/mautic/mautic/blob/3.3/app/bundles/EmailBundle/Form/Type/DashboardEmailsInTimeWidgetType.php#L62

Reply:
I found out why this was removed. It's duplicated in the file, so we're good to remove it here.